packages/react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -176,7 +176,8 @@ Checkbox.create = createShorthandFactory({
 })
 
 /**
- * A single checkbox within a checkbox group.
+ * A Checkbox allows a user to make choice from the set of two options.
```

Comment:
`set of two options` sounds a bit confusing to me, but it may only to me :)
I like this definition which I found `... make a choice between one of two possible mutually exclusive options`. Choose what you like more

Reply:
Fully agree that `mutually exclusive` is important to add to the description- would it be fine if we'll end up with this variant: `make a choice from two mutually exclusive options`?

Reply:
yes, perfect!